src/containers/BooksList.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+import { removeBook } from '../actions/index';
+
+class BooksList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleRemoveBook = this.handleRemoveBook.bind(this);
+  }
+
+  handleRemoveBook(book) {
+    const { removeBook } = this.props;
+    const s = this.props;
+    const bookIndex = (s.books).findIndex(x => x === book);
```

Comment:
why isn't `books` destructured from props just like `removeBook` is? :thinking:

Reply:
because on remove I am slicing based on the bookIndex